build/args/all.gn

```diff
@@ -2,6 +2,9 @@ is_electron_build = true
 use_jumbo_build = true
 root_extra_deps = [ "//electron" ]
 
+# Registry of NMVs --> https://github.com/nodejs/node/pull/24114/files
+node_module_version = 71
```

Comment:
we are on v8 7.5 in master, but this number doesn't matter in master. Maybe we should have a gclient hook that pulls the right number from that json registry if one is available, which might be useful sanity check when we cut branches,

Reply:
This number shouldn't be `71`, it is in direct conflict with node.  We should request another number in that JSON registry and set this to that number.

Reply:
Requested here: https://github.com/nodejs/node/pull/27288